docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3284,17 +3287,19 @@ Limitations:
 
 **Age Miswriting** 
 
-To implement this, first select the strings eligible for noise. For each 
-selected string, the age will be adjusted. The adjustment value will be 
-randomly selected from the user inputted list of possible perturbations. 
+To implement this, first select the rows for noise according to the row noise probability.
+For each selected row, the age will be adjusted. The adjustment value will be 
+randomly selected from the configured list of possible perturbations,
+according to the configured probabilities of selection. 
 
-For example, if the correct age is 28 and the possible perturbations are [-2, -1, 1, 2] 
-then 28 will be adjusted to either: 26, 27, 29, or 30, with an equal chance 
-of each option. 
+For example, if the correct age is 28 and the possible perturbations are [-2, -1, 1, 2]
+with configured probabilities of [0.2, 0.3, 0.3, 0.2]
+then 28 will be adjusted to either: 26, 27, 29, or 30, with a 0.2 probability for each of
```

Comment:
We better floor all of the ages in the post-processed data to make this make sense.